components/menu-dropdown/index.jsx

```diff
@@ -4,6 +4,7 @@
 // ### onClickOutside
 // Listen for clicks that occur somewhere in the document, outside of the element itself
 import onClickOutside from 'react-onclickoutside';
+import LanguageDirection from '../utilities/direction/private/language-direction';
 import MenuDropdown from './menu-dropdown';
 
-export default onClickOutside(MenuDropdown);
+export default LanguageDirection(onClickOutside(MenuDropdown));
```

Comment:
`MenuDropdown` doesn't need to know about `LanguageDirection` since alignment is flipped by `Dialog` now.